com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDEditorUtils.cs

```diff
@@ -282,6 +282,26 @@ internal static void HandlePrefixLabelWithIndent(Rect totalPosition, Rect labelP
         internal static bool IsPresetEditor(UnityEditor.Editor editor)
         {
             return (int)((editor.target as Component).gameObject.hideFlags) == 93;
+        }?
+        internal static void QualitySettingsHelpBox(string message, MessageType type, HDRenderPipelineUI.Expandable uiSection, string propertyPath)
+        {
+            if (CoreEditorUtils.HelpBoxWithButton(message, type))
+            {
+                SettingsService.OpenProjectSettings("Project/Quality/HDRP");
+                HDRenderPipelineUI.Inspector.Expand((int)uiSection);
+                Highlighter.Highlight("Project Settings", propertyPath, HighlightSearchMode.Identifier);
+                GUIUtility.ExitGUI();
+            }
+        }
```

Comment:
I would highly recommend to create a generic method in core.

something like: 

```
public static void DrawHelpBoxWithHighligt(string message, MessageType type, CoreEditorDrawer<T>, int section, string panelPath, string propertyPath)
```

And implement them here with the `Quality/HDRP` and `Graphics/HDRP Global Settings`

Reply:
if it's specific to HDRP i am not sure we should put it in core ? i propose to leave it here for now and see what can be shared once URP support is added